### DISCUSSION TITLE
时间效率为什么这么低

### DISCUSSION THREAD

**Discussion by fjw1049** [RapidAI/TableStructureRec#30] [Q&A]
为什么我的服务器跑的很慢

**Comment by fjw1049** [comment]
我采用gpu版本ocr发现较cpu慢了一半，是怎么回事，

**Reply by Joker1212** [reply]
emmm,主要耗时其实在ocr阶段，如果要使用gpu的话，可以参考在faq里面的链接，每次执行表格识别时，填入ocr_result,应该是rapid_ocr默认版本是针对cpu特别优化的，对gpu反而不够优化
<img width="836" alt="image" src="https://github.com/user-attachments/assets/8a7398d2-c1db-47d4-b250-f8ea1fdb1f17">

<img width="641" alt="image" src="https://github.com/user-attachments/assets/35cdd8c0-5b2e-4374-aeed-bd63a9c4c274">

**Comment by SWHL** [comment]
推荐用`rapidocr_paddle`gpu版，不推荐用`rapidocr_onnxruntime`GPU版，因为会慢很多。
原因参见：[link](https://rapidai.github.io/RapidOCRDocs/blog/2022/09/24/onnxruntime-gpu%E6%8E%A8%E7%90%86/)
